### ISSUE TITLE
module 'tensorflow' has no attribute 'placeholder' error

### ISSUE THREAD

**Issue by monistan** [Unisound/SampleRNN#19]
Hi, 
I keep getting stuck on this error when I try to run train.py
What is going on and how can I fix this?
ELI5 please because I'm new at this.

File "train.py", line 557, in <module>
    main()
  File "train.py", line 388, in main
    silence_threshold=args.silence_threshold)
  File "~/audio_reader.py", line 64, in __init__
    self.sample_placeholder = tf.placeholder(dtype=tf.float32, shape=None)
AttributeError: module 'tensorflow' has no attribute 'placeholder'

**Comment by jiamliang** [comment]
https://better-coding.com/solved-attributeerror-module-tensorflow-has-no-attribute-placeholder/
